mdsuite/calculators/structure_factor.py

```diff
@@ -332,13 +327,18 @@ def run_calculator(self):
             )
         total_structure_factor_li = np.array(total_structure_factor_li)
 
-        data = {"q": self.Q_arr.tolist(), "s(q)": total_structure_factor_li.tolist()}
+        data = {
+            self.result_series_keys[0]: self.Q_arr.tolist(),
```

Comment:
variables should be lower case if they are not constants.

Reply:
done